Calculator: Milestone2

brugobi opened the pull request:
### Implementatios

- As required for the Milestone 2:

- [X] Implement the App component;
- [X] Implement the Display component;
- [X] Implement the Button component;
- [X] Implement the ButtonPanel component;

## Thank you for the review

brugobi committed:
add components files

brugobi committed:
add App component

brugobi committed:
add componentes:button, buttonpanel, display

krys2fa commented on src/components/App.js:
```diff
@@ -0,0 +1,12 @@
+import ButtonPannel from './ButtonPanel';
+
+const App = () => (
+  <div>
+    <Display />
+    <ButtonPannel />
+  </div>
```
- [x] [REQUIRED] It would be better to have the children components wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/)) just like you did in the ButtonPanel component. This is because this helps you group a list of children without adding extra nodes to the DOM.

brugobi replied:
I added now
Thank you for reviewing

krys2fa reviewed (changes requested):
## Nice work @brugobi 👏 
Your project makes good use of functional components and ES6 destructuring assignment. 💪 
### Status: Changes Required ♻️ 

Please see the comment below for the requested change.

Remember to request another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

brugobi committed:
component App wrapped in React.Fragment

teekaytech reviewed (changes requested):
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @brugobi :wave:

__Thank you for implementing the required changes by the last reviewer.
However, there seems to be a bug that requires fixing before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] A bug is stopping your app from compiling. Kindly check the image below and fix it as appropriate.
![bug](https://user-images.githubusercontent.com/26856491/110825477-5fdcab00-8294-11eb-9fab-aef9a1a238be.png)

- [x] Kindly push your changes to the production server.

**[ OPTIONAL CHANGES ]**



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

brugobi committed:
change file name

brugobi commented:
- I updated to production and the live link It is working;
- I changed the name app.css to App.css and my computer is compiling with no error, even before changing the name, so I am not sure if the issue is fixed;

Thank you for the review

Torres-ssf reviewed (approved):
### Milestone Approved :+1: :clap: 
Hi @brugobi :hand: 
Awesome job on this milestone :smile: 
Good luck with the next one.

Happy coding :computer: 
**Sérgio Torres**
